sdk/remoterendering/tests.yml

```diff
@@ -4,6 +4,7 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: remoterendering
+      Location: westus2
```

Comment:
FYI @mikeharder the only other options seem to be south central or east us regions, which are further from the agent VM location (https://docs.microsoft.com/en-us/azure/remote-rendering/reference/regions). I'm more inclined to use westus2, but it might be worth trying the live tests in another US region to see if they work ok.

Reply:
Unless and until we hear otherwise, westus2 is fine with me for the limited scenarios that require it.